cla_backend/apps/cla_eventlog/tests/test_find_and_delete_old_cases.py

```diff
@@ -33,15 +34,24 @@ def create_case(self, current_time, case_type="legalaid.case"):
         freezer.stop()
         return case
 
-    def create_event_log_for_case(self, case, code, created):
-        make_recipe("cla_eventlog.log", case=case, code=code, created=created)
+    def create_event_log_for_case(self, case, code, created, created_by=None):
+        if created_by:
```

Comment:
I would recommend making a dict for these args
Something like:
```
kwargs = {case:case, code:code, created:created}
if created_by:
    kwargs.append(created_by)
make_recipe("cla_eventlog.log", **kwargs)
```

Reply:
Done. Had to change it to `kwargs.append(created_by=created_by)`